docker/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Waiting for database..."
+while ! python -c "import socket; s = socket.socket(); s.connect(('db', 5432))"; do
+  sleep 1
+done
+echo "Database is ready."
+
+python forum/manage.py makemigrations
```

Comment:
Repeatedly running `makemigrations` may result in unnecessary migrations or errors if no changes exist.
Check for unapplied migrations instead:
```
if python forum/manage.py showmigrations --plan | grep '\[ \]'; then
  python forum/manage.py migrate --fake-initial
else
  echo "No migrations to apply."
fi
```